src/types.ts

```diff
@@ -92,7 +92,8 @@ export type LastActiveTool =
         | "eraser"
         | "hand"
         | "frame"
-        | "embeddable";
+        | "embeddable"
+        | "mermaid";
```

Comment:
the mermaid popup should use `appState.openDialog` as it's not a tool (it'll also make it not persist across reloads which is the correct behavior). Was there any reason why use made it into a tool (apart from the fact it's in the toolbar)?

Reply:
Yes the only reason to put it under tools was its in the toolbar